packages/store/test/StoreCore.t.sol

```diff
@@ -68,12 +69,21 @@ contract StoreCoreTest is Test, StoreMock {
     bytes32 table2 = keccak256("other.table");
     IStore(this).registerSchema(table, schema, defaultKeySchema);
 
-    StoreCore.hasTable(table);
-
-    StoreCore.hasTable(table2);
-
     assertTrue(StoreCore.hasTable(table));
     assertFalse(StoreCore.hasTable(table2));
+
+    StoreCore.getSchema(table);
+    StoreCore.getKeySchema(table);
+
+    vm.expectRevert(
+      abi.encodeWithSelector(IStoreErrors.StoreCore_TableNotFound.selector, table2, TableId.toString(table2))
+    );
+    StoreCore.getSchema(table2);
+
+    vm.expectRevert(
+      abi.encodeWithSelector(IStoreErrors.StoreCore_TableNotFound.selector, table2, TableId.toString(table2))
+    );
+    StoreCore.getKeySchema(table2);
```

Comment:
I think this might have weird behaviour because StoreCore is a lib, not a contract, and `expectRevert` might pass even if only 1 of them actually reverts. Might have changed since last time I came across this though

Reply:
this test seems to pass as-is 🤷‍♂️

Reply:
yeah that should work

before this change, to see the problem I meant:
remove the revert from `getKeySchema`,
see that tests still pass (they shouldn't),
remove
```
vm.expectRevert(
   abi.encodeWithSelector(IStoreErrors.StoreCore_TableNotFound.selector, table2, TableId.toString(table2))
 );
StoreCore.getSchema(table2);
```
(so that there's only 1 expectRevert in the test),
see that test doesn't pass now

Reply:
basically you wouldn't notice this problem by tests failing, but rather by tests not failing when they should